src/usb.h

```diff
@@ -17,6 +17,9 @@
 #ifndef BL_USB_H
 #define BL_USB_H
 
+#define BL_STR_MANUFACTURER	"Codethink"
+#define BL_STR_PRODUCT		"Medical Plethysmograph Device"
```

Comment:
Might be worth moving the serial number here too, since it keeps it all together.

Also, if you use spaces for alignment instead of tabs, it will work whatever people have their tab step is set to.

Reply:
+1